app/feature/feature-edit-coinsured/src/main/kotlin/com/hedvig/android/feature/editcoinsured/ui/AddCoInsuredBottomSheetContent.kt

```diff
@@ -226,6 +233,37 @@ private fun FetchFromSsnFields(
   }
 }
 
+private fun personNummerInputFormatter(text: AnnotatedString, mask: String): TransformedText {
```

Comment:
I think this can just be a private class here instead right? So it's just created once. Can also change the name to just be the English version. It'll help with if we want to search for it in the future.
`private class PersonalNumberInputFormatter`

Reply:
Actually, can this be a class extending from `VisualTransformation` instead? And just pass an instance of that class at the call site above, without even a lambda? The name can also change to have `VisualTransformation` as a suffix instead

Reply:
I should've written inheriting, what you did is exactly what I had in mind. It's my deep engrained Java brain here I think, where we used `extends` instead of `inherits` when inheriting from abstract classes 😅

Reply:
Got it:) I thought about extension function and couldn't figure it out:))